src/builtins.ts

```diff
@@ -2844,7 +3333,18 @@ function deferASM(
   valueType: Type,
   reportNode: Node
 ): ExpressionRef {
-  var prototype = assert(compiler.program.elementsLookup.get(name));
```

Comment:
`elementsLookup` should also contain namespace members here, like both `Atomic` and `Atomic.load` etc. Is there a bug?

Reply:
I will cross check. `elementsLookup` was not in `elementsLookup` but it was a few months ago.

Reply:
For some reason namespace members are `undefined` in `elementsLookup`
```typescript
compiler.program.elementsLookup.get("Atomic.load")
```
return `undefined`

---------------
```typescript
compiler.program.elementsLookup.get("Atomic")
```
return 
```typescript
MapIterator {
  'load',
  'store',
  'add',
  'sub',
  'and',
  'or',
  'xor',
  'xchg',
  'cmpxchg',
  'wait',
  'notify' }
```